serve.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+if (($# != 2)); then
+  echo "Usage: $0 /tmp/logs localhost:8080" >&2
+  exit 1
+fi
+
+INDEXER_OUTPUT_DIR=$1
+
+# Serve the index
+# ===============
+# It's probably more efficient to cat them together, but this way we see
+# if a given one is corrupted for any reason.
+for e in "$INDEXER_OUTPUT_DIR"/*entries
+do
+  echo " * ${e}"
+  /opt/kythe/tools/write_entries --graphstore "$INDEXER_OUTPUT_DIR/gs" < "$e"
```

Comment:
You probably need to empty the `gs` and `tbl` dirs, so the new entries are not merged into.